user-guide/Advanced_Modules/Dashboards_app/Using_dashboard_feeds/Configuring_GQI_feeds/Configuring_an_external_data_source_in_a_query.md

```diff
@@ -168,6 +183,24 @@ In addition, the following derived types are supported from DataMiner 10.3.0/10.
 - `GQIStringDropdownArgument`
 - `GQIStringListArgument`
 
+### GQIDMS
+
+Available from DataMiner 10.3.4/10.4.0 onwards.
+
+The `GQIDMS` class contains the following methods, which can be used to request information in the form of `DMSMessage` objects:
+
+| Method | Function |
+|--------|----------|
+| `DMSMessage SendMessage(DMSMessage message)` | Sends a request that expects a single response. |
+| `DMSMessage[] SendMessages(params DMSMessage[] messages)` | Sends multiple requests at once, or sends a request that expects multiple responses. |
+
+The `GQIDMS` object is only generated when the property is used.
```

Comment:
This implementation detail is probably not really important for the official docs and might just cause confusion. I only wanted to mention it for the internal docs.